energy-integration/governance-v2/src/lib.rs

```diff
@@ -285,14 +323,16 @@ pub trait GovernanceV2:
                 );
             }
             GovernanceProposalStatus::Defeated => {}
+            GovernanceProposalStatus::WaitingForFees => {
+                // TODO: Dacă propunerea e defeated, dăm fee-ul înapoi?
```

Comment:
Nope, they should all be burned.

Reply:
Please delete comment. Use comment in English always.

Reply:
Fees do not have to be burnt, but we keep them in the contract. Delete the associated storage for that FEE.